.github/workflows/php-cs-fixer.yml

```diff
@@ -0,0 +1,25 @@
+name: PHP-CS-Fixer
+
+on:
+    pull_request:
+    push:
+
+jobs:
+    php-cs-fixer:
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 7.3
+                    coverage: none
+                    tools: php-cs-fixer:~2.16.0
```

Comment:
let's skip this and install https://github.com/php-coveralls/php-coveralls/blob/master/dev-tools/composer.json . that way, we don't need to specify tool version in 2 places

Reply:
This is draft - I planned to remove dev-tool folder at all. That looks like very old style desigion

Reply:
Purpose of dev-tools is that one can run all the tools on local machine.
Have `dev-tools`, use `shim`s in main `composer.json` or move to `PHIVE`.

Replacement of approach is OK, removal of tools and having only possibility to run them via CI or need for manual CI reverse engineering to local machine is no go

Reply:
This sound strange for me - I was never seen before something like this in open-source. 
Maybe add this directive into `require-dev` of the main `composer.json`?

Reply:
If you want to keep `dev-tools` folder I decide to use that and change script to run `composer sca`

Reply:
if you want to hear more why not putting dev-tools to require-dev, take a look: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4810#issuecomment-594101990
and similar topics.

in nutshell - you don't want to worry about cross-compatibility of your tools, as each of them may want same deps, but in different version (eg which of deps you put in require-dev are compatible with Sf6? it at least one is not, you cannot install Sf6)